MSAL/src/native_auth/network/responses/validator/reset_password/MSALNativeAuthResetPasswordResponseValidator.swift

```diff
@@ -83,7 +83,7 @@ final class MSALNativeAuthResetPasswordResponseValidator: MSALNativeAuthResetPas
             } else {
                 return .error(.invalidRequest(message: apiError.errorDescription))
             }
-        case .invalidClient:
+        case .unauthorizedClient:
             return .error(.invalidClient(message: apiError.errorDescription))
```

Comment:
I'd take the chance to rename all the internal references of "invalidClient" and rename them to "unauthorizedClient". The only endpoint that still handle "invalid_client" is the "/token", still in that specific case I'd rename the output error of the validator class to unauthorized_client.

Reply:
Code updated and updated description and name of PR to reflect change